webpack.config.js

```diff
@@ -19,6 +19,8 @@ module.exports = {
     path: path.resolve(__dirname, "dist/client"),
     publicPath: "/client",
   },
+  stats: "minimal",
+  mode: "development",
```

Comment:
I think we probably don't want to force the mode to development here - I imagine it would affect how the production build goes.

Reply:
good point!